src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/tests/burdenEstimateRepository/UploadBurdenEstimateTests.kt

```diff
@@ -0,0 +1,342 @@
+package org.vaccineimpact.api.databaseTests.tests
```

Comment:
this class has been split in 2 on master - could you work out what new content was added in this branch and add it to the appropriate new class (and then remove this file?)